src/Main.java

```diff
@@ -1,5 +1,64 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
+        //Задача1
+        System.out.println("Задача1");
+        for (int i = 1; i <= 10; i++) {
+            System.out.println(i);
+        }
+        //Задача2
+        System.out.println("Задача2");
+        for (int i = 10; i >= 1; i--) {
+            System.out.println(i);
+        }
+        //Задача3
+        System.out.println("Задача3");
+        for (int i = 0; i <= 17; i = i + 2) {
+            System.out.println(i);
+        }
+        //Задача4
+        System.out.println("Задача4");
+        for (int i = 10; i >= -10; i--) {
+            System.out.println(i);
+        }
+        //Задача5
+        System.out.println("Задача5");
+        for (int i = 1904; i <= 2096; i = i + 4) {
+            System.out.println(i + " год является високосным.");
+        }
+        //Задача6
+        System.out.println("Задача6");
+        for (int i = 7; i <= 77; i = i + 7) {
```

Comment:
```suggestion
        for (int i = 7; i <= 98; i = i + 7) {
```
до 98 по заданию нужно